### DISCUSSION TITLE
Header and Search Bar Discussion

### DISCUSSION THREAD

**Discussion by ghost** [colinwilson/lotusdocs#23] [General]
## Header

The name of the website should be displayed between the logo and the hamburger menu. This pattern is seen in many websites.

---

## Search Bar

The width of the search bar seems inconsistent:

1. It's smaller than the width of the left menu pane
2. It's also not as wide as the list item box (eg. the "Overview" box on [this page](https://lotusdocs.dev/docs/), or the "Features box on [this page](https://lotusdocs.dev/docs/guides/)

IMO the search bar should probably be in the left menu pane because: 

1. Breadcrumbs are configurable and ToC is also highlighted. So as far as single page content goes, there's not much scope to search.
3. Search makes more sense in the left menu pane as it supports nesting to second (and third ?) level and there could be a lot content.

**Comment by colinwilson** [comment]
> The name of the website should be displayed between the logo and the hamburger menu. This pattern is seen in many websites.

Yep. Will update this once I've decided on the font for Lotus Docs. Will also update the theme to allow for text next to the Logo mark for users who wish to have the website name as HTML text rather than a logo image.

> The width of the search bar seems inconsistent

This is on my To-Do list. Thanks for pointing this out. 👍 

> IMO the search bar should probably be in the left menu pane because...

Thanks for the suggestion but I prefer to keep the search button where it is for now. There are a few features I'm currently working on that will not be compatible with moving the search bar to the sidebar pane.

**Reply by ghost** [reply]
> Yep. Will update this once I've decided on the font for Lotus Docs. Will also update the theme to allow for text next to the Logo mark for users who wish to have the website name as HTML text rather than a logo image.

This is a really good approach !

> Thanks for the suggestion but I prefer to keep the search button where it is for now. There are a few features I'm currently working on that will not be compatible with moving the search bar to the sidebar pane.

Makes sense !
